### PR TITLE
Convert edit error page to TS

### PR DESCRIPTION
**Pull request by jonatanschroeder**

### CODE REVIEW

File: apps/prairielearn/src/pages/editError/editError.html.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      <body>
+        ${renderEjs(import.meta.url, "<%- include('../partials/navbar'); %>", {
+          ...resLocals,
+          navPage: 'error',
+        })}
+
+        <main id="content" class="container">
+          <script>
+            $(function () {
+              const button = document.querySelector('#results-some-uuid-button');
```
Comment (nwalters512) [inline apps/prairielearn/src/pages/editError/editError.html.ts:37]: I know this wasn't your change, but I'm confused why this is called `#results-some-uuid...`. Does this name make sense to you? I'd consider changing this.

File: apps/prairielearn/src/pages/editError/editError.html.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+              </div>
+            </h5>
+            <div class="card-body">
+              <p>The file edit did not complete successfully.</p>
+              ${failedSync
+                ? html`
+                    <p>
+                      In particular, it looks like your changes were written to
+                      disk${config.fileEditorUseGit
+                        ? 'and were pushed to the remote GitHub repository'
```
Comment (nwalters512) [inline apps/prairielearn/src/pages/editError/editError.html.ts:72]: I think you're missing a space here.

```suggestion
                        ? ' and were pushed to the remote GitHub repository'
```

File: apps/prairielearn/src/pages/editError/editError.html.ts (hunk 1 above)
Comment (jonatanschroeder) [reply apps/prairielearn/src/pages/editError/editError.html.ts:37]: This likely comes from the instructor file editor page, which has the `results` button include a UUID, likely from copying code from the pl-file-editor. In #10233 I removed that UUID stuff (see https://github.com/PrairieLearn/PrairieLearn/pull/10233/commits/c592fcaedb95fcbb57a0eb0829768b52192816eb). I can do the same changes here as well for consistency.

Comment (jonatanschroeder) [reply apps/prairielearn/src/pages/editError/editError.html.ts]: Renamed in 5191a74. Also took the opportunity to replace the click event with bootstrap events to clean it up.

**Review by nwalters512 — APPROVED** [review]

**Pull request merged**